.github/workflows/linkchecker.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+on: push
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      # Check out the latest version of the code
+      - uses: actions/checkout@v3.5.2
+
+      # Checks the status of hyperlinks in *.md files in docs/
+      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.13
+        with:
+          folder-path: "docs"
```

Comment:
Its a bit odd to use docs / docs, but beyond the scope of this PR to look into it

Reply:
Given that we have .md files in docs, we should make sure that everything there works. Otherwise we should just remove it. For example, docs/old/testing.md contains info that we refer to from the main README.md.

Reply:
Now that I think about it, we should probably check README.md also for broken links.